src/Tests/addedFunctionality.test.js

```diff
@@ -0,0 +1,76 @@
+/* @jest-environment jsdom*/
+/* eslint-disable */
```

Comment:
Please, remember also not to disable linter checks in-line. Linters represent coding style guidelines that you share with your team and altering the configurations or making efforts to by-pass them might lead to divergent coding styles in a codebase, thereby messing it up.

Discipline yourself now and you'll be glad you did.